mephisto/providers/mturk/mturk_utils.py

```diff
@@ -326,14 +327,21 @@ def convert_mephisto_qualifications(
         if converted["Comparator"] is None:
             converted["Comparator"] = qualification["comparator"]
 
-        if converted["IntegerValue"] is None and converted["LocaleValues"] is None:
+        if (
+            converted["IntegerValue"] is None
+            and converted["IntegerValues"] is None
+            and converted["LocaleValues"] is None
+        ):
             value = qualification["value"]
             if isinstance(value, list):
                 converted["IntegerValues"] = value
+                del converted["IntegerValue"]
```

Comment:
should this be the plural key being deleted...

Reply:
The idea here is only one of `IntegerValues`, `IntegerValue`, or `LocaleValues` should be set. As such, if it's a list, we delete the singular, and if it's the singular we delete the list.